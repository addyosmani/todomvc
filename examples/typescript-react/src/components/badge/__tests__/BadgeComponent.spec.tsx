import React from 'react';
import { render } from '@testing-library/react';

import { BadgeComponent } from '../BadgeComponent';

describe('<BadgeComponent />', () => {
	it('should render badge', async () => {
		const { container } = await render(<BadgeComponent name='label' />);
		expect(container.firstChild).toMatchSnapshot();
	});

	it('should display text', async () => {
		const testLabel = 'some-label';
		const { findByText } = await render(<BadgeComponent name={testLabel} />);
		expect(findByText(testLabel)).toBeTruthy();
	});
});
