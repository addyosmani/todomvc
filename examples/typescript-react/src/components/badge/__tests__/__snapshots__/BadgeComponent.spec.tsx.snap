// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BadgeComponent /> should render badge 1`] = `
<div
  class="sc-bdnxRM flBqrq"
>
  label
</div>
`;
